### DISCUSSION TITLE
v2.61.1

### DISCUSSION THREAD

**Discussion by nunocoracao** [nunocoracao/blowfish#1333] [Announcements]
## What's Changed
* ⚙️ Update params.toml by @nunocoracao in https://github.com/nunocoracao/blowfish/pull/1318
* 🎏 added translation lib and scripts by @nunocoracao in https://github.com/nunocoracao/blowfish/pull/1320
* ⚙️ Update Hugo supported version by @github-actions in https://github.com/nunocoracao/blowfish/pull/1323
* 🎏 update id.yaml by @fauziralpiandi in https://github.com/nunocoracao/blowfish/pull/1328
* 🐛 Tailwind Colors are not updating with colorScheme by @nunocoracao in https://github.com/nunocoracao/blowfish/pull/1331
* 🔖 v2.61.1 by @nunocoracao in https://github.com/nunocoracao/blowfish/pull/1317

**Full Changelog**: https://github.com/nunocoracao/blowfish/compare/v2.61.0...v2.61.1

<hr /><em>This discussion was created from the release <a href='https://github.com/nunocoracao/blowfish/releases/tag/v2.61.1'>v2.61.1</a>.</em>

**Comment by aakashnand** [comment]
I am leaving my thoughts here about the new look of the blowfish site. 

The background animation is a bit distracting to the user.
